nose_ignores.txt

```diff
@@ -32,4 +32,5 @@
 --ignore-file=test_stream_particles\.py
 --ignore-file=test_stream_stretched\.py
 --ignore-file=test_version\.py
+--ignore-file=test_gadget_pytests\.py
```

Comment:
I guess the file should be renamed (`pytests` -> `pytest`)

Reply:
because `test_gadget_pytests.py` only includes a single test? Or did you have another reason?

Reply:
that's just because `pytest` is spelled without an trailing `s`. There's also a convention that some "pytest-only" test modules are named `*_pytest.py` (usually the suffix is only used to avoid name collisions with other modules)

Reply:
ah, got it. happy to change it!